uarray/_uarray_dispatch.cxx

```diff
@@ -338,7 +382,7 @@ struct SetBackendContext
       return 0;
     }
 
-  static PyObject * enter__(SetBackendContext * self, PyObject * /*args*/)
+  static PyObject * enter__(SetBackendContext * self)
```

Comment:
This is wrong, see the [docs](https://docs.python.org/3/c-api/structures.html#c.PyCFunction)
> `PyCFunction`
    Type of the functions used to implement most Python callables in C. Functions of this type take two `PyObject*` parameters and return one such value.

Reply:
Interesting, I was following this: https://pythonextensionpatterns.readthedocs.io/en/latest/parsing_arguments.html#no-arguments